tools/serverpod_cli/lib/src/generator/dart/library_generators/model_library_generator.dart

```diff
@@ -74,6 +73,10 @@ class SerializableModelLibraryGenerator {
             from: p.dirname(classDefinition.filePath),
           );
           libraryBuilder.directives.add(Directive.partOf(topNodePath));
+        } else if (!(classDefinition.isSealedTopNode &&
+            classDefinition.childClasses.isEmpty)) {
+          libraryBuilder.directives
+              .insert(0, Directive.import('package:meta/meta.dart'));
```

Comment:
Question: Why the `classDefinition.isSealedTopNode` check here? I think an explicit test for this case would be nice!

Reply:
There was some special logic to see when to add the import directive for `meta`, which must only be added if a `copyWith` was being implemented. Otherwise consumers would get a warning for an unused import.

But then I changed it so that `@useResult` is exported by the Serverpod libs, so that we don't get that warning on using `meta` in the generated code (which might not be a dependency of the consuming package). Now we don't need any check anymore, as the "Serverpod import" is available in all files.